package.json

```diff
@@ -29,7 +29,8 @@
     "object-sizeof": "^1.6.1",
     "prettier": "^2.3.0",
     "rate-limiter-flexible": "^2.3.6",
-    "stream-json": "^1.7.3"
+    "stream-json": "^1.7.3",
+    "supports-color": "^8"
```

Comment:
also, `supports-color` 9 requires ESM, which we aren't using. so, like oclif, we're going to use this older version.

oclif doesn't expose the result of "supports-color", so we need to depend on this separately.